processor/src/queues.ts

```diff
@@ -50,11 +51,49 @@ const processIBOMQueue = new bullmq.Queue('processIBOM', {
 })
 projectQueues.push(processIBOMQueue)
 
-async function createJobs(jobData: JobData) {
+const cleanUpQueue = new bullmq.Queue('cleanUp', {
+  connection: redisConnection,
+  defaultJobOptions,
+})
+
+const repoProcessingFlow = new FlowProducer({ connection: redisConnection })
+
+/**
+ * Create jobs for each project in a repo, and when all projects are done, clean up the checkout and the output dir.
+ */
+async function createJobs(jobData: RepoJobData) {
   const jobId = jobData.outputDir
-  for (const q of projectQueues) {
-    await q.add('projectAPI', jobData, { jobId })
-  }
+
+  const childrenJobs = jobData.kitspaceYamlArray
+    .map(kitspaceYaml => {
+      const projectOutputDir = path.join(jobData.outputDir, kitspaceYaml.name)
+      kitspaceYaml.summary = kitspaceYaml.summary || jobData.repoDescription
+      const data: ProjectJobData = {
+        ...jobData,
+        kitspaceYaml,
+        outputDir: projectOutputDir,
+        subprojectName: kitspaceYaml.name,
+      }
+
+      return projectQueues.map(q => ({
+        data,
+        name: 'projectAPI',
+        opts: { jobId: projectOutputDir },
+        queueName: q.name,
+      }))
+    })
+    .flat()
+
+  await repoProcessingFlow.add({
+    children: childrenJobs,
```

Comment:
Grammatically correct would be `childJobs`.

Reply:
Done!